integration-tests/typescript-angular/src/generated/api.github.com.yaml/client.service.ts

```diff
@@ -15450,9 +15447,7 @@ export class ApiClient {
     owner: string
     repo: string
   }): Observable<
-    | (HttpResponse<t_interaction_limit_response | EmptyObject> & {
-        status: 200
-      })
+    | (HttpResponse<t_interaction_limit_response | any> & { status: 200 })
```

Comment:
Based on the input source being:
```
      description: Shows which type of GitHub user can interact with this organization
        and when the restriction expires. If there is no restrictions, you will see
        an empty response.
...
            application/json:
              schema:
                anyOf:
                - "$ref": "#/components/schemas/interaction-limit-response"
                - type: object
                  properties: {}
                  additionalProperties: false
```

I think that the intention of the github spec was for an `| EmptyObject` here - need to put a bit more effort into nailing down my understanding of exactly what the specification is here.

Reply:
Fixed in https://github.com/mnahkies/openapi-code-generator/pull/204/commits/a0223ec5b99efe2f1b3957ee569c89953e4aee50

There may still be some variations I've missed, but I think in general things are more correct than they were before this PR